packages/utils/src/types/accountEntry.ts

```diff
@@ -4,13 +4,33 @@ import { ethers } from 'ethers'
 import { u8aSplit, serializeToU8a, MULTI_ADDR_MAX_LENGTH, u8aEquals } from '..'
 import BN from 'bn.js'
 import { Address, PublicKey } from '.' // TODO: cyclic dep
+import { decode } from 'bs58'
+import { publicKeyVerify } from 'secp256k1'
 
 export class AccountEntry {
   constructor(
     public readonly address: Address,
-    public readonly multiAddr: Multiaddr,
+    public readonly multiAddr: Multiaddr | undefined,
     public readonly updatedBlock: BN
-  ) {}
+  ) {
+    // If we have a multiaddr
```

Comment:
is there an instance where we don't have a multiaddr? it's a required argument

Reply:
At least here https://github.com/hoprnet/hoprnet/blob/master/packages/utils/src/types/accountEntry.ts#L57